tests/kernel/sleep/src/sleep.c

```diff
@@ -214,25 +220,27 @@ void main(void)
 	k_wakeup(test_thread_id);
 
 	if (test_failure) {
-		goto done_tests;
+		zassert_true(0, "test failure");
```

Comment:
please remove if block. you do the same wit zassert_* as well.

Reply:
Done